src/main/java/com/softserveinc/ita/mmakoviichuk/pageobjects/softserve/SoftServePage.java

```diff
@@ -9,19 +9,16 @@
 
 public class SoftServePage {
 
-    private SelenideElement sidebarSection;
-
     public void switchSidebarSection (String section) {
-        sidebarSection = $x(format("//a[div[@class = 'side-navigation__title' and text() = '%s']]", section));
-        sidebarSection.click();
+        $x(format("//a[div[@Class = 'side-navigation__title' and text() = '%s']]", section)).click();
```

Comment:
move into class template variable -> //a[div[@Class = 'side-navigation__title' and text() = '%s']]